integr8ly/manifests
# add solution explorer 0.0.27

verification:
- have an openshift 4 cluster
- create an operatorsource pointing at the quay repo, `aidenkeating`
- create a namespace, e.g. intly-2346
- install the integreatly operator in that namespace
- create an installation
- ensure version 0.0.27 of the solution explorer operator is deployed
- ensure version v1.7.2 of the walkthroughs are deployed, in solution
  explorer deploymentconfig env
- ensure version 2.18.0 of the solution explorer image is deployed,
  in solution explorer deploymentconfig image
